packages/editor/src/components/blocks/select/Select.tsx

```diff
@@ -0,0 +1,45 @@
+import type { Prettify, Select } from '@axonivy/form-editor-protocol';
+import type { ComponentConfig, UiComponentProps } from '../../../types/config';
+import './Select.css';
+import { baseComponentFields, defaultBaseComponent } from '../base';
+import IconSvg from './Select.svg?react';
+import { IvyIcon } from '@axonivy/ui-components';
+import { IvyIcons } from '@axonivy/ui-icons';
+
+type SelectProps = Prettify<Select>;
+
+export const defaultInputProps: Select = {
+  label: 'Label',
+  value: '',
+  items: [],
+  itemsValue: '',
+  ...defaultBaseComponent
+} as const;
+
+export const SelectComponent: ComponentConfig<SelectProps> = {
+  name: 'Select',
+  category: 'Elements',
+  subcategory: 'Interactions',
+  icon: <IconSvg />,
+  description: 'A simple input with a label',
+  defaultProps: defaultInputProps,
+  render: props => <UiBlock {...props} />,
+  create: ({ label, value, ...defaultProps }) => ({ ...defaultInputProps, label, value, ...defaultProps }),
+  fields: {
+    label: { subsection: 'General', label: 'Label', type: 'text' },
+    value: { subsection: 'General', label: 'Value', type: 'textBrowser' },
+    items: { subsection: 'General', label: 'Options', type: 'selectTable' },
+    itemsValue: { subsection: 'General', label: 'Bind Options', type: 'textBrowser' },
```

Comment:
Me as a low coder has not clue what bind options mean.

Why not simply use the terms 'Static options' and 'Dynamic options'

Reply:
Yes thanks, I struggeld a bit with the naming but I think static and dynamic options is very good 👍